pkg/apis/hco/v1alpha1/hyperconverged_types.go

```diff
@@ -29,6 +29,9 @@ type HyperConvergedSpec struct {
 
 	// operator version
 	Version string `json:"version,omitempty"`
+
+	//configurations to be passed to kubevirt config-map
+	KubevirtConfiguration kubevirtv1.KubeVirtConfiguration `json:"kubevirtConfiguration,omitempty"`
```

Comment:
We are going to move to v1beta1 with https://github.com/kubevirt/hyperconverged-cluster-operator/pull/695

I'd suggest to do this only there.
We also have to properly think about the upgrade path for who already deployed in the past.

Reply:
Thank you @tiraboschi I'll try and sync with #695. 
About the second point, can you elaborate?
We are changing the default, but anyone who want's can change it back, it also stated for a long time the we 
recommend to use masquerade and not bridge. so at least in that case we are "coverd". aren't we?
@phoracek  what is your opinion about that second point?